content/en/docs/installation/supported-releases.md

```diff
@@ -0,0 +1,108 @@
+[v0.16]: https://cert-manager.io/docs/release-notes/release-notes-0.16
+[v0.15]: https://cert-manager.io/docs/release-notes/release-notes-0.15
+[v0.14]: https://cert-manager.io/docs/release-notes/release-notes-0.14
+[v0.13]: https://cert-manager.io/docs/release-notes/release-notes-0.13
+[v0.12]: https://cert-manager.io/docs/release-notes/release-notes-0.12
+[v0.11]: https://cert-manager.io/docs/release-notes/release-notes-0.11
+
+You can find available releases on the [releases
+page](https://github.com/cert-manager/cert-manager/releases). You can find
+the release notes for each minor release
+[here](https://cert-manager.io/docs/release-notes/), and the upgrade
+instructions are
+[here](https://cert-manager.io/docs/installation/upgrading/). The
+cert-manager release process is documented on the [release-process
+page](https://cert-manager.io/docs/contributing/release-process/).
+
+## Support policy
+
+In the below section, we refer
+
+We support the current release as well as the two last minor version
+releases of cert-manager, as shown in the above table. By _supported
+release_, we mean that the cert-manager team will:
+
+1. Produce patch releases for critical bugs on the supported releases.
+   Critical bugs are patched as soon as possible and do not follow the two
+   months regular schedule. Regressions and security fixes fall under the
+   critical bugs category.
+2. Offer technical assistance on a best-effort basis for these releases,
+   with the help of the community. Assistance can be asked on the
+   [Kubernetes Slack](https://slack.k8s.io/) (channel `#cert-manager`), as
+   a [GitHub Discussion](https://github.com/jetstack/cert-manager/discussions) or
+   using the
+   [cert-manager-dev](https://groups.google.com/g/cert-manager-dev) Google
+   group.
+
+Around once every two months, we build a minor release, e.g., `v1.4`,
+denoted `N` in the below table. Around the same time, we issue one backport
+patch release, e.g. `v1.3.1`, that we denote `N-1` in the below table. This
+backport patch release contains the bugs that we identified as "important
+bugs", such as bugs preventing the upgrade to an earlier version.
+
+Regular bugs will only be fixed on the next release of cert-manager (`N+1`
+in the below table).
+
+| Severity      | Examples             |  Is previous release patched?  |  Is current release patched?   | Or does it wait until next release? |
+| ------------- | -------------------- | :----------------------------: | :----------------------------: | :---------------------------------: |
+| Critical bug  | [#2857][], [#3393][] |       Yes, patched ASAP        |       Yes, patched ASAP        |                 No                  |
+| Important bug | [#3644][]            | Yes, released every two months | Yes, released every two months |                 No                  |
+| Regular bug   |                      |               No               |               No               |                 Yes                 |
```

Comment:
I don't think there is a clear distinction here between these 3, nor what constitutes what.

Can we simplify this a bit?

1. Security issue
2. Critical bug (functionality that *did* work in the release before but no longer works/something has broken)
3. Long-standing bugs (bugs that have been acknowledged and have been known for a long time, and there are well known workarounds)

I think (1) should trigger a patch release for _all_ supported releases as soon as the patch is ready

(2) should be back-ported, but not immediately trigger a release (i.e. a patch release will be made for all supported releases according to the regular release schedule)

(3) I think is probably the hardest to define case - I think longstanding bugs typically have been around for a while because the fix is _non trivial_ (i.e. requires some heavy refactoring work, or some redesign). These sorts of patches are usually _not_ so acceptable for backporting, as they introduce large numbers of LoC changed. In these instances, I think we should not backport the patches *at all* as it could risk the stability of user clusters due to their size/complexity

Reply:
Thank you for your comment, this is super useful!! Can I copy some of your comments into the page?  😇